app/src/components/Api.ts

```diff
@@ -0,0 +1,12 @@
+export const PRODUCTION = process.env.NODE_ENV === "production";
```

Comment:
All `.ts` files should start with a lowercase letter since they don't hold React components. This file should be `api.ts`